yousee-channel-archiving-requester-web/src/main/java/dk/statsbiblioteket/mediaplatform/ingest/channelarchivingrequester/web/YouSeeChannelMappingRESTServlet.java

```diff
@@ -160,14 +164,17 @@ public String addChannel(@FormParam("ycm_channel") String channel,
             if (fromDate == null || "".equals(fromDate)) {
                 fromDate = "1900-01-01";
             }
-            newFromDate = ZonedDateTime.parse(fromDate);
+            LocalDate localFromDate = LocalDate.parse(fromDate, dateFormatter);
+            newFromDate = localFromDate.atStartOfDay(localZone);
             ycm.setFromDate(Date.from(newFromDate.toInstant()));
 
             if (toDate == null || "".equals(toDate)) {
                 toDate = "3000-01-01";
             }
-            newToDate = ZonedDateTime.parse(toDate);
-            if (newToDate.isAfter(ZonedDateTime.ofInstant(ycm.getFromDate().toInstant(), localZone)) || newToDate.equals(ycm.getFromDate()))
+            LocalDate localToDate = LocalDate.parse(toDate, dateFormatter);
+            newToDate = localToDate.atStartOfDay(localZone);
+
+            if (newToDate.isAfter(ZonedDateTime.ofInstant(ycm.getFromDate().toInstant(), localZone)) || newToDate.equals(newFromDate))
```

Comment:
Instead of "ZonedDateTime.ofInstant(ycm.getFromDate().toInstant(), localZone)" we should be able to just use newFromDate? (Seeing as we just set it)